Caboodle/Compass/Compass.uwp.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Threading;
+using Windows.Devices.Sensors;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class Compass
+    {
+        internal static bool IsSupported =>
+            Windows.Devices.Sensors.Compass.GetDefault() != null;
```

Comment:
😜 
Maybe refactor this so you don't have to do it twice?

Reply:
#resolved